api/app/main.py

```diff
@@ -356,7 +356,7 @@ def stats_demo(
     features = _calculate_stats(
         zones_filepath,
         geotiff,
-        stats=" ".join(["min", "max", "mean", "median", "sum", "std"]),
+        stats=" ".join(DEFAULT_STATS),
```

Comment:
You've dropped "sum" and "std" in doing this. did you mean to?

Reply:
nice catch! fixed